datahub-web-react/src/app/onboarding/onboarding.tsx

```diff
@@ -0,0 +1,693 @@
+                    </a>
+                </p>
+            </Typography.Paragraph>
+        ),
+    },
+    {
+        id: USERS_ASSIGN_ROLE_ID,
+        selector: `#${USERS_ASSIGN_ROLE_ID}`,
+        title: 'Assigning Roles',
+        content: (
+            <Typography.Paragraph>
+                <p>
+                    You can also assign <strong>Roles</strong> to existing <strong>Users</strong> here.
+                </p>
+                <p>
+                    Learn more about <strong>Roles</strong>{' '}
+                    <a
+                        target="_blank"
+                        rel="noreferrer noopener"
+                        href="https://datahubproject.io/docs/authorization/roles"
+                    >
+                        {' '}
+                        here.{' '}
+                    </a>
+                </p>
+            </Typography.Paragraph>
+        ),
+    },
+    {
+        id: GROUPS_INTRO_ID,
+        title: 'Groups Page',
+        content: (
+            <Typography.Paragraph>
+                <p>
+                    Welcome to the <strong>Groups</strong> page!
+                </p>
+                <p>
+                    <strong>Groups</strong> are collections of users which can be used to assign ownership to assets and
+                    configure authorization.
+                </p>
+                <p>
+                    <strong>Groups</strong> can be created natively via the DataHub UI, or synced from your Identity
+                    Provider.
+                </p>
+                <p>
+                    Learn more about <strong>Groups</strong>{' '}
+                    <a
+                        target="_blank"
+                        rel="noreferrer noopener"
+                        href="https://datahubproject.io/docs/authorization/groups"
+                    >
+                        {' '}
+                        here.{' '}
+                    </a>
+                </p>
+            </Typography.Paragraph>
+        ),
+    },
+    {
+        id: GROUPS_CREATE_GROUP_ID,
+        selector: `#${GROUPS_CREATE_GROUP_ID}`,
+        title: 'Create a new Group',
+        content: (
+            <Typography.Paragraph>
+                <p>
+                    Click here to create a new <strong>Group</strong>.
+                </p>
+            </Typography.Paragraph>
+        ),
+    },
+    {
+        id: ROLES_INTRO_ID,
+        title: 'Roles Page',
+        content: (
+            <Typography.Paragraph>
+                <p>
+                    Welcome to the <strong>Roles</strong> page!
+                </p>
+                <p>
+                    DataHub currently supports three out-of-the-box Roles: <strong>Admin</strong>,{' '}
+                    <strong>Editor</strong> and <strong>Reader</strong>.
+                </p>
+                <p>
+                    <strong>Roles</strong> are the recommended way to manage permissions on DataHub.
```

Comment:
This sentence should come first logically, right? 

DataHub Roles

Roles are the recommended way....